### PR TITLE
enh(driver):support schemalessraw from  V1.0.8

### PR DESCRIPTION
**Pull request by xleili**

### CODE REVIEW

File: examples/OptsTelnetRaw.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                $"{table} 1648432611249 221 location=北京.顺义 groupid=4",
+                $"{table} 1648432611250 217 location=北京.顺义 groupid=4",
+            };
+        string selectSql = $"select * from {table}";
+        public void RunOptsTelnetRaw(IntPtr conn)
+        {
+            Console.WriteLine("RunOptsTelnetRaw ...");
+
+            int rows = TDengine.SchemalessInsertRaw(conn, lines,TDengineSchemalessProtocol.TSDB_SML_TELNET_PROTOCOL,TDengineSchemalessPrecision.TSDB_SML_TIMESTAMP_NOT_CONFIGURED);
+            Console.WriteLine("SchemalessInsertRaw using TSDB_SML_TELNET_PROTOCOL insert {0} rows...", rows);
```
Comment (sangshuduo) [inline examples/OptsTelnetRaw.cs:26]: Should we check if insert failed?

File: examples/Schemaless.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        public void RunSchemaless(IntPtr conn)
+        {
+            UtilsTools.ExecuteUpdate(conn, "create database if not exists sml_db");
+            UtilsTools.ExecuteUpdate(conn, "use sml_db");
+
+            this.Line.RunInfuxDBLine(conn);
+            this.JSON.RunOptsJSON(conn);
+            this.Telnet.RunOptsTelnet(conn);
+            UtilsTools.ExecuteUpdate(conn, "drop database if exists sml_db");
+
```
Comment (sangshuduo) [inline examples/Schemaless.cs:28]: extra space

File: examples/SchemalessRaw.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        public void RunSchemalessRaw(IntPtr conn)
+        {
+            UtilsTools.ExecuteUpdate(conn, $"create database if not exists {DB} keep 3650");
+            UtilsTools.ExecuteUpdate(conn, $"use {DB}");
+
+            this.Line.RunInfluxDBLineRaw(conn);
+            this.optsJSONRaw.RunOptsJSONRaw(conn);
+            this.optsTelnetRaw.RunOptsTelnetRaw(conn);
+            UtilsTools.ExecuteUpdate(conn, $"drop database if exists {DB}");
+
```
Comment (sangshuduo) [inline examples/SchemalessRaw.cs:30]: extra space

File: examples/InfluxDBLineRaw.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            Console.WriteLine("SchemalessInsertRaw using InfluxDBLine insert {0} rows...", rows);
+            //check insert
+
+            IntPtr res = UtilsTools.ExecuteQuery(conn, selectSql);
+            //check insert
+            UtilsTools.DisplayRes(res);
+
+            //free res
+            TDengine.FreeResult(res);
+
```
Comment (sangshuduo) [inline examples/InfluxDBLineRaw.cs:36]: extra space

File: examples/InfluxDBLineRaw.cs [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        string[] lines =  {
+                $"{table},location=California.LosAngeles,groupid=2 current=11.8,voltage=221,phase=0.28 1648432611249",
+                $"{table},location=Ca\0l0ifornia.LosAngeles,groupid=2 current=13.4,voltage=223,phase=0.29 1648432611250",
+                $"{table},location=Ca\\0lifornia.LosAngeles,groupid=3 current=10.8,voltage=223,phase=0.29 1648432611249",
+                $"{table},location=北京\0.朝阳,groupid=3 current=11.0,voltage=220,phase=0.36 1648432611251",
+                $"{table},location=北京.顺义,groupid=3 current=11.1,voltage=220,phase=0.35 1648432611252"
+            };
+        string selectSql = $"select * from {table}";
+
+
```
Comment (sangshuduo) [inline examples/InfluxDBLineRaw.cs:19]: extra space

File: examples/InfuxDBLine.cs [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        string[] lines =  {
+                "influx_line,location=California.LosAngeles,groupid=2 current=11.8,voltage=221,phase=0.28 1648432611249",
+                "influx_line,location=California.LosAngeles,groupid=2 current=13.4,voltage=223,phase=0.29 1648432611250",
+                "influx_line,location=California.LosAngeles,groupid=3 current=10.8,voltage=223,phase=0.29 1648432611249",
+                "influx_line,location=California.LosAngeles,groupid=3 current=11.3,voltage=221,phase=0.35 1648432611250"
+            };
+        string selectSql = "select * from influx_line";
+
+
```
Comment (sangshuduo) [inline examples/InfuxDBLine.cs:17]: extra space

File: examples/InfuxDBLine.cs [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+            //check insert
+
+            res = UtilsTools.ExecuteQuery(conn, selectSql);
+            //check insert
+            UtilsTools.DisplayRes(res);
+
+            //free res
+            TDengine.FreeResult(res);
+
```
Comment (sangshuduo) [inline examples/InfuxDBLine.cs:45]: extra space

**Review by sangshuduo — COMMENTED** [review]
some comments

File: examples/OptsTelnetRaw.cs (hunk 1 above)
Comment (xleili) [reply examples/OptsTelnetRaw.cs:26]: SchemalessInsertRaw will throw exception,  if  insert failed.
Check has been done in SchemalessInsertRaw.
`            try
            {

                IntPtr smlRes = _taos_schemaless_insert_raw(taos, utf8Bytes, utf8Bytes.Length, totalRowsPtr, (int)protocol, (int)precision);
                if (ErrorNo(smlRes) != 0)
                {
                    throw new Exception($"{Error(smlRes)},{ErrorNo(smlRes)}");
                }
                return Marshal.ReadInt32(totalRowsPtr);
            }
            finally
            {
                Marshal.FreeHGlobal(totalRowsPtr);
            }`

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**